src/dss/main.py

```diff
@@ -30,5 +31,43 @@ def initialize_command(kubeconfig: str) -> None:
     initialize(lightkube_client=lightkube_client)
 
 
+@main.command(name="logs")
+@click.option(
+    "--kubeconfig",
+    help=f"Path to a Kubernetes config file. Defaults to the value of the KUBECONFIG environment variable, else to '{KUBECONFIG_DEFAULT}'.",  # noqa E501
+)
+@click.argument("notebook_name", required=False)
+@click.option(
+    "--all", "print_all", is_flag=True, help="Print the logs for all notebooks and MLflow."
+)  # noqa: E501
+@click.option("--mlflow", is_flag=True, help="Print the logs for the MLflow deployment.")
+def logs_command(kubeconfig: str, notebook_name: str, print_all: bool, mlflow: bool) -> None:
+    """
+    Prints the logs for the specified notebook or DSS component.
+    """
```

Comment:
What about a more verbose doc string? I didn't understand how exactly this worked until I read the whole function including the examples.

Reply:
Thats the help message we agreed with UX team in the spec.

Reply:
Is the doc string also being used in what the user sees?